### ISSUE TITLE
Configuration files for deploying to the cloud

### ISSUE THREAD

**Issue by veeral-patel** [karlkeefer/pngr#12]
Hi! I just saw your HN post. You're right -- being able to just run `docker-compose up` makes me a lot more inclined to start developing something new.

However, I'm reluctant to deploy my application to production using Docker. For the frontend I'd like to use something like Netlify for caching/web UI and for the backend I'd like to use something like App Engine for monitoring/scaling/etc

I feel like if you added an App Engine and Netlify configuration file, so you could clone the project  and deploy to "production" in 2 commands, that would move this project to the next level :)

**Comment by karlkeefer** [comment]
I originally was thinking of setting up a CI pipeline and leaving the actual deploy destination "up to the user", but I can see the value in including some recipes for common deployment destinations...

Will think on this.

**Comment by karlkeefer** [comment]
@veeral-patel I agree this is useful but I'm not likely to work on it as my production path doesn't involve either of these. Most of my projects just get little $5/mo VPS which is considerably cheaper than even the smallest GCP instance.

I will definitely merge something in if you (or someone) provides some reasonable configs for this!
